tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -172,3 +179,37 @@ Check GPU Resources
         Page Should Not Contain Element     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
     END
 
+Verify Quick Starts Work As Expected When All the Steps Marked As Yes
+    [Arguments]     ${element}
+    Open Element By Name    ${element}
+    ${count}=   Get The Count Of Description
+    FOR     ${index}    IN RANGE    ${count}
+        Move To Next Description By Clicking Yes
+    END
+    Verify Previous Description
+    Move To Next Description By Clicking Yes
+    Verify The Progress Of The Items    ${element}  Complete
+
+Verify Quick Starts Work As Expected When Restarting The Previous One
+    [Arguments]     ${element}
+    Element Text Should Be  //article[@id="${element}"]//a      Restart
+    Open Element By Name    ${element}
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+    Page Should Contain Element     //div[@data-testid="qs-drawer-close"]/button
+    Click Button        //div[@data-testid="qs-drawer-close"]/button
+    CLick Button        //button[@data-test="cancel button"]
+    Page Should Contain Element     //div[@class="pf-c-drawer__panel-main"]
+    Click Button        //div[@data-testid="qs-drawer-close"]/button
+    Click Button        //button[@data-test="leave button"]
+    Element Text Should Be  //article[@id="${element}"]//a      Open
```

Comment:
not sure about this. I think it should click on "continue" rather than "open". After this, it should check if the the restarted QS starts from where it was left as reported in ODS-1166

Reply:
news on that?